app.py

```diff
@@ -39,15 +39,16 @@ def _annotate(self, mmif: Mmif, **parameters) -> Mmif:
         self.mmif = mmif if isinstance(mmif, Mmif) else Mmif(mmif)
         self.text_doc = self.mmif.get_documents_by_type(DocumentTypes.TextDocument)
         assert len(self.text_doc) == 1, "There should be exactly one TextDocument in the MMIF file"
+        label_set = set(parameters["containLabel"])
+        assert len(label_set) >= 1, "There should be at least one label in the parameters"  # Here we make the label parameter as required
```

Comment:
SDK will do check this when nothing passed for a "required" parameter. To make a parameter required, just take out `default=` argument when setting a parameter up. 

*(SDK will do the check, but due to this bug https://github.com/clamsproject/clams-python/issues/232 , the output will be messy for now. )